### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by normanrz**
- Adds the new WEBKNOSSOS logo
- Changes the primary color to #5660FF
- Changes brand name from webKnossos to WEBKNOSSOS

### URL of deployed dev instance (used for testing):
- https://rebranding.webknossos.xyz

### Steps to test:
- Click around on https://rebranding.webknossos.xyz

------
(Please delete unneeded items, merge only when none are left open)
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Updated [documentation](../blob/master/docs) if applicable
- [x] Ready for review

### CODE REVIEW

File: frontend/stylesheets/dark.less [hunk 1]
```diff
@@ -1,6 +1,10 @@
 @import "../../node_modules/antd/dist/antd.dark.less";
 @import "./main.less";
 
+body {
+  background-color: #000;
```
Comment (philippotto) [inline frontend/stylesheets/dark.less:5]: To quote from the Refactoring UI pdf:

> True black tends to look pretty unnatural, so start with a *really* dark grey and work your way up to white in steady increments.

I think the same goes for a white background. Maybe you feel like toning these down a tiny notch.

Comment (normanrz) [reply frontend/stylesheets/dark.less:5]: This just fixes an inconsistency (see below). Tuning the background is a bigger job. 
Screenshot 2023-01-13 at 12 00 37

**Comment by fm3** [comment]
Favicon seems to be #6456ff, not #5660FF, is that intended?

**Comment by normanrz** [comment]
> Favicon seems to be #6456ff, not #5660FF, is that intended?

That shouldn't be the case. Where are you seeing that?

**Comment by jstriebel** [comment]
Just found this gray text which I found hard to read in dark-mode, it's okayish in light-mode (might not even be affected by the new colors):
Screenshot from 2023-01-13 13-34-39
Screenshot from 2023-01-13 13-37-09

**Comment by jstriebel** [comment]
I'm not quite sure if Nunito is used correctly, I couldn't find it in the sources. Maybe move it to fonts, next to the Titilium woff2 files?

**Comment by fm3** [comment]
> Where are you seeing that?

image

(favicon purple-ish, big svg bluer)

**Comment by philippotto** [comment]
Regarding the new font: it still doesn't seem to be used on my machine. Or is this expected?

image

**Comment by normanrz** [comment]
For me it loads and uses the network font, now.
Screenshot 2023-01-13 at 15 08 00

**Comment by philippotto** [comment]
I found some spots where old colors are still used:

- in the dataset folder view the left sidebar's background is still the old blue when dragging a dataset
- the active entries in the tree list and segment list view in an annotation

The rest is looking good :+1:

**Comment by normanrz** [comment]
> The only thing I noticed is that the margins now seem different for multi-line tags. Not sure which change caused this but on master, this doesn't seem to be the case:
> 
> image
> 
> new vs vs
> 
> image
> 
> But this doesn't need to block the PR in my opinion.

How about this?
Screenshot 2023-01-17 at 09 45 03

**Comment by philippotto** [comment]
> How about this?

Yes, this looks good :+1:

**Pull request merged**